### DISCUSSION TITLE
Academic affiliation was not verified, but second email says I am verified in global campus?

### DISCUSSION THREAD

**Discussion by Cheespeasa1234** [community/community#56794] [GitHub Education] [answered]
### Hi everyone,

I recently sent in my ID to get student benefits. I later got a rejection email for multiple reasons, such as it being modified. However, I recently got an email welcoming me to the campus, and the bottom of the email said I am verified? Am I good to go or not?

**Comment by mrepol742** [comment, marked as answer]
I think you are good to go since you have `PRO` Highlights in your profile. You can check if its approved here https://education.github.com/discount_requests/application

**Comment by samus-aran** [comment]
Hey @Cheespeasa1234  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
